Project.toml

```
@@ -19,7 +19,7 @@ VideoIO = "d6d074c3-1acf-5d4c-9a43-ef38773959a2"
 [compat]
 BenchmarkTools = "0.5"
 CSV = "0.6, 0.7, 0.8"
-CUDA = "1.1, 2.0, 2.1, 2.2, 2.3"
+CUDA = "1.1, 2.0, 2.1, 2.2, 2.3, 3.0"
```

Why this isn't the equivalent
```suggestion
CUDA = "1.1, 2.0, 3.0"
```
?